csharp/FAudio.cs

```diff
@@ -27,6 +27,7 @@
 #region Using Statements
 using System;
 using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
```

Comment:
Which type needed this namespace?

Reply:
This is probably just VS2022 being incredibly stupid, it loves to add random using statements now